frappe/calendar_map.json

```diff
@@ -0,0 +1,16 @@
+{
+	"Event":{
```

Comment:
Formatting, there should be a space before `{`
```
"Event": {
```

Reply:
done